task-runner/tests/unit/test_task_request_handler.py

```diff
@@ -75,6 +75,7 @@ def download_input_side_effect(
 
     task_request_payload = {
         "sim_dir": "sim_dir",
+        "run_subprocess_dir": None,
```

Comment:
I thought about it, but the user interface would get too convoluted.

We might implement something more thought out in the future. but right now this was just to make fvcom work.

Reply:
I believe this would need more consideration.
I could add it. But we would have Command("command1", run_subprocess_dir="./run")
The run_subprocess_dir is in relation with the input dir. Like this we are creating a strange dependency from Command and the input_dir.

I would need to think more about this.